public/video-ui/src/actions/WorkflowActions/getProdOffice.js

```diff
@@ -0,0 +1,26 @@
+import WorkflowApi from '../../services/WorkflowApi';
+
+function receiveProdOffice(prodOffice) {
+  return {
+    type: 'WORKFLOW_PRODOFFICE_GET_RECEIVE',
+    receivedAt: Date.now(),
+    prodOffice
+  };
+}
+
+function errorReceivingProdOffice(error) {
+  return {
+    type: 'SHOW_ERROR',
+    message: `Could not get Workflow production office. <a href="${WorkflowApi.workflowUrl}" target="_blank" rel="noopener">Open Workflow to get a cookie.</a>`,
+    error: error,
+    receivedAt: Date.now()
+  };
+}
+
+export function getProdOffice() {
+  return dispatch => {
+    return WorkflowApi.getProdOffice()
```

Comment:
did we actually create this function? seems like this file is left over from when we thought we were fetching the prod office stuff from the workflow API (which doesn't actually supply it)

Reply:
oh thanks joe! i should have put a message on this. Didn't get to finish it and I DO need to add this function